pkg/graph/schema.graphql

```diff
@@ -53,6 +53,35 @@ type CedarSystem {
 	systemMaintainerOrgComp: String
 }
 
+"""
+"""
+type CedarSystemBookmark {
+  euaUserId: String!
+  cedarSystemId: UUID!
+}
+
+"""
+The data needed to bookmark a cedar system
+"""
+input CreateCedarSystemBookmarkInput {
+  cedarSystemId: UUID!
+}
+
+"""
+The payload when bookmarking a cedar system
+"""
+type CreateCedarSystemBookmarkPayload {
+  cedarSystemBookmark: CedarSystemBookmark
+  userErrors: [UserError!]
```

Comment:
I've been trying to dig deeper and figure out the purpose of `userErrors` in some of these other schema. It seems to be there to handle errors that propogate to the UI, though I wonder why it would make sense to return the `Payload` with an error in it, rather than just returning an empty payload with an `error` as the second return value (as all these resolvers return an error as their second type)

Thoughts?

Reply:
Yeah, I'm not sure why that value is even in payloads at all.  Ideally the FE wouldn't even attempt to send the payload if there were errors with it.  It seems to be in all the payload schemas.

Reply:
Yeah when the FE gets a response, it usually checks that second return for the error, not inside the data payload.

Reply:
Yeah - @jstrang-oddball maybe we can just remove this property if we're not gonna be using it, and if we find a need for it we can add it back!

Reply:
Will go ahead and remove that.

Reply:
d4f1c3110a12d0f18388d54c470850d74ba7abc3